node/node.go

```diff
@@ -1379,3 +1379,9 @@ func (node *AlgorandFullNode) VotingKeys(votingRound, keysRound basics.Round) []
 func (node *AlgorandFullNode) Record(account basics.Address, round basics.Round, participationType account.ParticipationAction) {
 	node.accountManager.Record(account, round, participationType)
 }
+
+// IsParticipating implements network.NodeInfo
+func (node *AlgorandFullNode) IsParticipating() bool {
+	round := node.ledger.Latest() + 1
+	return node.accountManager.HasLiveKeys(round, round)
```

Comment:
would it make sense to broaden the range here to start getting some transactions flowing in advance of the round when your key comes online? e.g. (latest+1, latest+10) or something?

Reply:
Added 10 rounds into the future. Current txn pool is only 3 rounds, but a little extra sounds future-proof-ish for now.